### ISSUE TITLE
Spinners on dark background need better contrast ratio

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3527]
## Describe the bug

On the spinners page of the website, the dark background example fails constract ratios.

## How to reproduce

https://clarity.design/documentation/spinners

- Locate "Spinners on a dark background" example
- Observe that the foreground color is #0079B8 and the background color is #313131
- Use the Color Contrast Analyzer to check the contrast ratio of these colors (https://developer.paciellogroup.com/resources/contrastanalyser/) and observe that #0079b8 and #0079b8 yield a 2.7:1 contrast ratio

## Expected behavior

Use colors that yield a contrast ration of at least 3:1.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

Since Clarity cannot determine the background color of the container for the spinner, we can make a note on our website about this and fix our demo to have proper contrast.

**Comment by mathisscott** [comment]
Implementation:

The inverse spinner needs to use the action-blue from the dark theme. Namely, #49AED9.

This is only for the inverse spinner. It would be nice if these were tied to SASS variables. Presently, the inverse spinner always seems to show up with the #0079b8 color, no matter what theme is involved.

**Comment by AlexMarcuBytex** [comment]
@mathisscott hello, was this issue solved ? i found that the inverse spinner is using a plain #74c1e2 hex color. Let me know if i can help with this issue.
`background: generateSpinnerIcon($clr-spinner-inverse-bg-color, #74c1e2);`

**Comment by mathisscott** [comment]
@AlexMarcuBytex 
Yes, thanks for pointing this out. I completed this as part of #3286 and did not find this issue in github when i did so.

This is fixed.
